applications/tari_console_wallet/src/automation/commands.rs

```diff
@@ -637,6 +649,18 @@ pub async fn command_runner(
                     Err(e) => eprintln!("BurnTari error! {}", e),
                 }
             },
+            CreateKeyPair(args) => match create_key_pair(key_manager_service.clone(), args.key_seed).await {
```

Comment:
The extra function seems unnecessary. I would also name the arg `key_branch` or `branch_name`
```suggestion
            CreateKeyPair(args) => match key_manager_service.create_key_pair(args.key_branch).await {
```

Reply:
Much better ! Thanks